pkg/ccl/cloudccl/externalconn/testdata/multi-tenant/privileges_show_external_connections

```diff
@@ -0,0 +1,52 @@
+subtest usage-privilege-on-ec
```

Comment:
Can we put this test in `pkg/sql/logictest/testdata/logic_test/external_connection_privileges ` instead?